README.md

```diff
@@ -1,2 +1,270 @@
+
+TBD: The fixed use of the period type in the `effective` property originates in HealthConnectOnFHIR. Should we also allow DateTime for instant observations?
+
+#### Compound Observations
+
+|code>system|code>code|code>display|value|components|
+|-|-|-|-|-|-|
+|"http://loinc.org"|"85354-9"|"Blood pressure panel with all children optional"|-|"Diastolic blood pressure", "Systolic blood pressure"|
+
+#### Simple Observations
+
+|code>system|code>code|code>display|value>value|value>code|value>unit|
+|-|-|-|-|-|-|
+|"http://loinc.org"|"55423-8"|"Number of steps"|double|?|"steps"|
+|"http://loinc.org"|"91557-9"|"Lean body weight"|double|"[lb_av]"|"g" or "lbs"|
+|"http://loinc.org"|"29463-7"|"Body weight"|double|"[lb_av]"|"g" or "lbs"|
+|"http://loinc.org"|"41981-2"|"Calories burned"|double|?|"kcal"|
+|"http://loinc.org"|"8302-2"|"Body height"|double|?|"m"|
+|"http://loinc.org"|"8867-4"|"Heart rate"|double|?|"beats/minute"|
+|"http://loinc.org"|"8310-5"|"Body temperature"|double|?|"°C"|
+|"http://loinc.org"|"8462-4"|"Diastolic blood pressure"|double|"mm[Hg]"|"mmHg"|
+|"http://loinc.org"|"8480-6"|"Systolic blood pressure"|double|"mm[Hg]"|"mmHg"|
+
+TODO: Currently, HealthKitOnFHIR and HealthConnectOnFHIR still use different units to encode the same observations. We should probably aim to always use the same unit to reduce unit conversions, but each client would need to be able to handle multiple units for each observation type. We should probably aim for SI units, even though the system will primarily be used in the US. TBD.
+
+[Codes/Units in HealthKitOnFHIR](https://github.com/StanfordBDHG/HealthKitOnFHIR/blob/main/Sources/HealthKitOnFHIR/Resources/HKSampleMapping.json)
+
+### /users/$userId$/questionnaireResponses/$questionnaireResponseId$
+
+Based on [FHIR QuestionnaireResponse](https://hl7.org/fhir/R4B/questionnaireresponse.html), the following properties may be used, while additional properties are ignored by the Engage-HF system.
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|id|string|-|[Resource](https://hl7.org/fhir/R4B/resource.html): Logical id of this artifact|
+|questionnaire|string|-|canonical representation of the questionnaire, i.e. t|
+|author|string|-|The patient's id|
+|source|string|-|The patient's id|
+|authored|DateTime|-|The date the answers were gathered.|
+|status|[QuestionnaireResponseStatus](https://hl7.org/fhir/R4B/valueset-questionnaire-answers-status.html)|-|Will most likely always be `completed`.|
+|item|list of Item|-|-|
+|item[x]>linkId|string|-|Pointer to specific item from questionnaire|
+|item[x]>definition|optional uri|-|details for item|
+|item[x]>text|optional string|-|Name for group or question text|
+|item[x]>answer|list of Answer|-|response(s) to the question|
+|item[x]>answer[y]>value|any|-|Value depending on the type of question in the survey, e.g. boolean, integer, date, string, etc|
+
+TBD: Should we use `source` and/or `patient` to store a reference to the patient?
```

Comment:
Would make sense, I would suggest to store the user identifier here.

Reply:
It was more of a question on which one to use. I will check how ResearchKitOnFHIR handles it.

Reply:
It doesn't 😂 It simply doesn't set the value.

Reply:
Yes, we should make this configurable, I think that we had that option at one point and should re-introduice it. Would be gerat to create an issue for this in the repo.

Reply:
I changed it to be author for now - if we decide to change it later, we can simply adapt the document.